components/Base/FileUpload.vue

```diff
@@ -42,8 +42,32 @@ withDefaults(
 const emit = defineEmits(['update:file']);
 
 const fileInput = ref();
+const isError = ref(false);
 const selectedFile = ref<File | undefined>();
 
+const inputClasses = computed(() => {
+  return isError.value ? 'input-error' : 'input-default';
+});
+const inputMessage = computed(() => {
+  return isError.value
+    ? 'Incorrect file type uploaded. Drop a correct file type.'
+    : selectedFile.value?.name;
+});
+
+const validateFileType = (loadFiles: FileList): File | undefined => {
+  const acceptedList = props.accept.split(',');
+  const loadFile = loadFiles[0];
+  if (
+    props.accept === '.*' ||
+    acceptedList.some((i) => loadFile.name.includes(i))
```

Comment:
Imagine we have `.jpg` on our `acceptedList`, but we didn't include `.torrent`. Wouldn't that fail if someone would use for example `not_a_torrent.jpg.torrent`? 🤔

Also, I have even further going example, look at this:
<img width="344" alt="image" src="https://user-images.githubusercontent.com/66744401/228282460-4610ea2e-e555-472d-a367-8c52c4e97aff.png">

I don't think file name provides a respective information about the file type.

Reply:
I believe all methods here can be easily circumvented if someone want's to be malicious, even if we enforce 'mime' types or even by `application`

Reply:
But it's not only for on purpose malicious actions. I believe this is an important part of UX to get informed that "hey, sorry, maybe you did not do this intentionally, but your file is invalid" 🤔

Reply:
Anyways my point is to keep that stuff in mind. I don't think we don't have to really be focused on that particular thing in this project 😄